src/base/bittorrent/session.cpp

```diff
@@ -583,7 +583,7 @@ void Session::setLSDEnabled(const bool enabled)
     {
         m_isLSDEnabled = enabled;
         configureDeferred();
-        LogMsg(tr("Local Peer Discovery support [%1]").arg(enabled ? tr("ON") : tr("OFF"))
+        LogMsg(tr("Local Peer Discovery (LSD) support: %1").arg(enabled ? tr("ON") : tr("OFF"))
```

Comment:
It is `Local Service Discovery` according to BEP.

Reply:
Almost everywhere on the net use "Local Peer Discovery" and even here: https://www.libtorrent.org/features-ref.html
Maybe we should replace `(LSD)` with `(LPD)` instead?

Reply:
>Maybe we should replace (LSD) with (LPD) instead?

And it will create some inconsistencies in the code base (especially webapi)... maybe it is not worth it?

Reply:
> And it will create some inconsistencies in the code base (especially webapi)... maybe it is not worth it?

OK, forget it.
I just drew attention to the inconsistency of the abbreviation with its decoding. Maybe just use one thing in the user visible text?